### PR TITLE
[Docs]: Fix Metadata Reference 

### PR DESCRIPTION
**Pull request by eaba**
## Changes

Turn on `ProduceReferenceAssembly`

### CODE REVIEW

**Review by Aaronontheweb — CHANGES_REQUESTED** [review]
Just do this in `common.props`

**Comment by Aaronontheweb** [comment]
Delete all of your `output.txt` files as well.

**Comment by eaba** [comment]
> Delete all of your `output.txt` files as well.

Skipped this too, apology!

**Comment by Aaronontheweb** [comment]
No problem @eaba

**Comment by eaba** [comment]
@Aaronontheweb done!

**Review by Aaronontheweb — CHANGES_REQUESTED** [review]

File: src/contrib/cluster/Akka.Cluster.Metrics/Akka.Cluster.Metrics.csproj [hunk 1]
```diff
@@ -8,6 +8,7 @@
             and to the registered subscribers on the system event bus with the help of Cluster Metrics Extension.
         </Description>
         <TargetFramework>$(NetStandardLibVersion)</TargetFramework>
+        <ProduceReferenceAssembly>$(ProduceReferenceAssembly)</ProduceReferenceAssembly>
```
[the remark this answers is not in the document]
Comment (eaba) [reply src/contrib/cluster/Akka.Cluster.Metrics/Akka.Cluster.Metrics.csproj:11]: We don't need that there right?

[the remark this answers is not in the document]
Comment (Aaronontheweb) [reply src/contrib/cluster/Akka.Cluster.Metrics/Akka.Cluster.Metrics.csproj:11]: Nope, `common.props` gives us a single place to manage these. We only need them in the F# projects for override.

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Pull request merged**